web/src/helpers.tsx

```diff
@@ -96,3 +97,12 @@ export const getUrlQuery: (key: string) => string | false = (key: string) => {
   }
   return false;
 };
+
+export const timestampToLocaleString = (timestamp: number) => {
+  if (!timestamp) {
+    // TODO: i18n
+    return "None"
```

Comment:
I think it's wired to show text `None` in the time column and some of user maybe don't know what's it mean. How about return empty string ``? 

![image](https://user-images.githubusercontent.com/31329157/98572188-4e888f80-22f0-11eb-9a75-c465d5d47e07.png)

Reply:
This is a issue from backend, please see the related issue.